nvdaHelper/readme.md

```diff
@@ -11,18 +11,37 @@ You should still build on the command line to verify errors.
 
 - Ensure you have built NVDA on the command line first.
 - Create a new project from existing code
-- Type: Visual C++
+- Type: Visual C++, press next.
 - Set the `<repo root>/nvdaHelper/` directory as the project file location.
 - Project name: "nvdaHelper"
 - Add files to the project from these folders: checked.
-- Other defaults are fine
-- Includes: `../include;../miscdeps/include;./;../build\x86_64;%(AdditionalIncludeDirectories)`
-- Defines: `WIN32;_WINDOWS;_USRDLL;NVDAHELPER_EXPORTS;UNICODE;_CRT_SECURE_NO_DEPRECATE;LOGLEVEL;_WIN32_WINNT;_WIN32_WINNT_WIN7;`
-- Force Includes: `winuser.h`
-- Create the project.
+  - This should have a single 'checked' item, the path to nvdaHelper
+- Other defaults are fine, press next
+- Select "use external build system" for "How do you want to build the project?", press next
+- Build command line: `scons source`
+- Include search paths: `../include;../miscdeps/include;./;../build\x86_64;%(AdditionalIncludeDirectories)`

```

Comment:
In the repository `miscDeps` are with a capital D. While Windows does not care about this it makes sense to be consistent IMO:
```suggestion
- Include search paths: `../include;../miscDeps/include;./;../build\x86_64;%(AdditionalIncludeDirectories)`
```